src/components/CcyInfoModal/CcyInfoModal.js

```diff
@@ -13,13 +13,33 @@ const CcyInfoModal = ({
   fetchCcyArticle,
   article,
 }) => {
+  const { body, title } = article
+
+  const containerRef = useRef()
   useEffect(() => {
     if (isModalVisible && _isEmpty(article)) {
       fetchCcyArticle()
     }
   }, [isModalVisible])
 
-  const { body, title } = article
+  useEffect(() => {
+    if (!body || !containerRef.current) {
+      return
+    }
+    const links = [...containerRef.current.querySelectorAll('a')]
+    links.forEach(link => {
+      const targetAttr = link.getAttribute('target')
+      const refAttr = link.getAttribute('rel')
+      if (targetAttr !== '_blank') {
+        link.setAttribute('target', '_blank')
+      }
+      if (refAttr !== 'noopener') {
+        link.setAttribute('rel', 'noopener')
+      }
+    })
+    console.log(links)
```

Comment:
this should be removed

Reply:
removed, https://github.com/bitfinexcom/bfx-hf-ui-core/pull/57